integration_test/login_test.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:star_movie_app/main_sandbox.dart' as app;
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  const splashScreenDelay = Duration(seconds: 3);
+
+  group('Login Test', () {
+    const validLogin = 'test@mail.ru';
+    const validPassword = 'test1234';
+
+    testWidgets('login auth test', (tester) async {
```

Comment:
is this ok to leave this big anonymous function inside testWidgets?

Reply:
I think yes